common/enablers/enableEpochsHandler_test.go

```diff
@@ -130,11 +134,10 @@ func TestNewEnableEpochsHandler_EpochConfirmed(t *testing.T) {
 
 		cfg := createEnableEpochsConfig()
 		handler, _ := NewEnableEpochsHandler(cfg, &epochNotifier.EpochNotifierStub{})
-		require.False(t, check.IfNil(handler))
+		require.NotNil(t, handler)
 
-		handler.EpochConfirmed(77, 0)
+		handler.EpochConfirmed(85, 0)
```

Comment:
when we will merge the master -> rc/v1.5.0 -> rc/v1.6.0 this line will become `handler.EpochConfirmed(math.MaxUint32, 0)`.